frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -682,6 +685,15 @@ export default function InteractiveWaveformModal({
   };
 
   const handleVersionSelect = (idx) => {
+    const selectedVersionMetadata = metaPulse3dVersions.filter(
+      (version) => version.version === pulse3dVersions[idx]
+    )[0];
+    setPulse3dVersionEOLDate(
+      selectedVersionMetadata.end_of_life_date
+        ? ` Version ${selectedVersionMetadata.version} will be removed after ${electedVersionMetaData.end_of_life_date}.`
+        : `Version ${selectedVersionMetadata.version} will be removed soon.`
```

Comment:
Also is this saying that every version without an EOL dat will be marked with 'Version x.xx.xx will be removed soon'? 
Probably could just be empty if we don't know. is a modal popping up every time any version is selected?

Reply:
Yes. So if there is a version that is deprecated but now end of life date is given the modal will default to removed soon. But if a end of life date is present it will put the date in the modal instead